### PR TITLE
Add issue template for tracking issues

### PR DESCRIPTION
**Pull request by seanchen1991**
## Description

Adds a new issue template for tracking issues.

______

### PR author checklist:
- [ ] Added changelog entry, using [`unclog`](https://github.com/informalsystems/unclog).
- [ ] Added tests: integration (for Hermes) or unit/mock tests (for modules).
- [ ] Linked to GitHub issue.
- [ ] Updated code comments and documentation (e.g., `docs/`).
- [ ] Tagged *one* reviewer who will be the one responsible for shepherding this PR.

### Reviewer checklist:

- [ ] Reviewed `Files changed` in the GitHub PR explorer.
- [ ] Manually tested (in case integration/unit/mock tests are absent).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/tracking-issue.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<!-- Add tasks/issues that contribute to this effort here -->
+
+- [ ] 
+
+____
+
+#### For Admin Use
+
+- [ ] Not duplicate issue
+- [ ] Appropriate labels applied
```
Comment (adizere) [inline .github/ISSUE_TEMPLATE/tracking-issue.md:21]: We should refine this expectation here. The "appropriate label" is probably a specific one in the case of tracking issues, right? Probably https://github.com/informalsystems/hermes/labels/admin or https://github.com/informalsystems/hermes/labels/okr ?

File: .github/ISSUE_TEMPLATE/tracking-issue.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- [ ] Appropriate milestone (priority) applied
+- [ ] Appropriate contributors tagged
+- [ ] Contributor assigned/self-assigned
```
Comment (adizere) [inline .github/ISSUE_TEMPLATE/tracking-issue.md:24]: I think we can get rid of this expectation

**Review by adizere — COMMENTED** [review]
Looks good, thanks Sean!

One important suggestion I have is for us to be consistent in what we mean by "tracking" vs. "OKR" vs. "admin" issue, to prevent confusion.

- We manage OKRs in the workflow-explorer (internal tool).
- We manage _admin_ and _tracking_ issues directly in this repo (and other repos).
    - It seems like _admin_ is a more vague term we use interchangeably with _tracking_. Is that right? 
    - If so, we should consolidate and have a single label/name, whether its admin/tracking.
    - If the 2 labels serve distinct purposes, we should clarify their respective meaning.

It seems like this template is the right place to clarify what we mean exactly by "tracking" and "admin" issues.

File: .github/ISSUE_TEMPLATE/tracking-issue.md (hunk 1 above)
Comment (seanchen1991) [reply .github/ISSUE_TEMPLATE/tracking-issue.md]: I added a note at the top of the template. Hopefully that makes it clear enough which label is appropriate for which situation.

File: .github/ISSUE_TEMPLATE/tracking-issue.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  _Note_: Tracking issues are meta-level issues meant to capture smaller units 
+  of work that contribute to a larger initiative. As such, tracking issues should
+  always be tagged with either the 'tracking' label or the 'okr' label. 
+  
+  The 'okr' label should be applied when the tracking issue maps directly with a quarterly
+  OKR activity. Additionally, the OKR-labeled issue should also be linked to in the 
+  Workflow Explorer. 
+
+  Not all tracking issues will map with OKRs. Such issues should thus be tagged with 
+  the 'tracking' label.
```
Comment (adizere) [inline .github/ISSUE_TEMPLATE/tracking-issue.md:16]: Super clear, thanks Sean!

**Review by adizere — APPROVED** [review]

**Pull request merged**